src/main/java/com/example/onlinebookstore/model/Book.java

```diff
@@ -26,4 +30,6 @@ public class Book {
     private BigDecimal price;
     private String description;
     private String coverImage;
+    @Column(nullable = false)
```

Comment:
Set column name “is_deleted”, it's better naming for tables

Reply:
(jpg|jpeg|png|gif|ico|bmp|svg|webp|tiff|gif|jpe|jfif)

Reply:
In video Bogdan had "isDeleted" field

Reply:
It's ok. Hibernate by default converting from camelCase to snake_case